src/script/view_model/ContentViewModel.ts

```diff
@@ -271,9 +271,6 @@ export class ContentViewModel {
         this.conversationState.activeConversation(conversationEntity);
       }
 
-      if (conversationEntity.is_cleared()) {
-        conversationEntity.cleared_timestamp(0);
-      }
```

Comment:
We do not mark conversation as not cleared on conversation open event. Only event appearing in the conversation should do that.